2022/project_idea.md

```diff
@@ -1,106 +1,116 @@
 
-To aid with Bayesian modeling analysis ArviZ provides both numerical and visual summaries,
-as well as a collection of software tools that help with modeling and storage of statistical results.
+To aid with Bayesian modeling analysis, ArviZ provides both numerical and visual summaries,
+as well as a collection of software tools that help with modeling and storing statistical results.
 
 ArviZ currently has a Julia and a Python interface and we are working on compatibility with R too.
 To date of writing, ArviZ integrates with at least 9 different Probabilistic Programming Languages (PPLs),
 including all major PPLs like Stan, PyMC or Turing.
 
-### Contact
-For any inquiry about Season of Docs at ArviZ, contact us at [gitter](https://gitter.im/arviz-devs/season_of_docs).
+### Contact 
+For any inquiry about Season of Docs at ArviZ, contact us at gitter.
 
 ## Project motivation
-Writing educational content to teach how to use ArviZ is hard, writing educational content to teach
-how to use PPLs is hard and writing content to teach how to use Bayesian modeling is also hard.
-However, the good news is that all these are not inherently hard because of some exceptional
-technical complexity. One huge challenge in writing such educational material is that
-all these tasks are actually one and the same, a wider task generally called
-[Bayesian workflow](https://arxiv.org/abs/2011.01808).
-
-Solving real problems with Bayesian modeling requires computational tools,
-so trying to explain Bayesian modeling without using any PPL is very limited.
-But at the same time, explaining how to use a PPL to someone who doesn't know
-anything about Bayesian modeling is also a fool's errand.
-And any modeling without visualization, diagnostics and model checking is
+Writing educational contents to teach how to use ArviZ  could pose a certain level of difficulty, producing educational contents to teach
+how to use PPLs and Bayesian modeling could equally pose a level of difficulty. This difficulty isn't because these frameworks and libraries are inherently complex or too technical but rather as a result of having an encompassing documentation that captures all
+the possible libraries that are used with the Bayesian workflow
+without doing a disservice to any group of users by not capturing their choice of library or language. 
+Here's the icebreaker, writing such educational material/guide is actually one and the same, 
+a wider task generally called Bayesian workflow.[Bayesian workflow](https://arxiv.org/abs/2011.01808].
+
+Solving real problems with the Bayesian model requires computational tools, 
+so trying to explain Bayesian modeling without using any PPL is very limited and not efficient. 
+But at the same time, explaining how to use a PPL to someone who doesn't know 
+anything about Bayesian modeling is also a fool's errand. 
+Any modeling without visualization, diagnostics and model checking is 
```

Comment:
```suggestion
Solving real problems with Bayesian modeling requires computational tools, 
so trying to explain Bayesian modeling without using any PPL is very limited and not efficient. 
But at the same time, explaining how to use a PPL to someone who doesn't know 
anything about Bayesian modeling is also a fool's errand. 
Moreover, any modeling without visualization, diagnostics and model checking (tasks done with ArviZ) is 
```

I have changed "the Bayesian model" with "Bayesian modeling", there isn't a single Bayesian model. "Bayesian modeling" is a way of understanding probability that allows statisticians to build any model and adapt them to the data
in order to get a good explanation of the observations.

This paragraph emphasizes with examples the general idea exposed in the paragraph above. I would then also move the ideas you suggested about the disservice to our users to a new paragraph right below this one. Let me know if this structure is making the idea we want to transmit a bit more clear.

Reply:
That's fine.

Reply:
> Thanks for the updates and for keeping the original lines heart it makes it much easier to review!
> 
> I have done some formatting changes that I'll commit myself. They will be marked as resolved, but you can still open them and see the changes and explanation in some of the cases. You'll have to pull if you are working locally.
> 
> Let me know if you don't know how to do it or would do a pair programming session to look at this or the reviews that need further discussion, you can reach me at writethedocs slack for example. I really appreciate you reviewing this proposal and making sure it is clear to people without any background on Bayesian statistics (like I expect all reviewers for where we submit this to try and get some funding to be)

I'm glad I could help, thank you.